velox/exec/HashTable.h

```diff
@@ -23,6 +23,11 @@
 
 namespace facebook::velox::exec {
 
+constexpr std::string_view kCapacity{"hashtable.capacity"};
```

Comment:
Please, add comments to explain that these are the names of the runtime stats reported by operators that use the HashTable.

Reply:
Fixed